nepal_compliance/nepal_compliance/report/sales_return_register_ird/sales_return_register_ird.py

```
@@ -70,45 +70,99 @@ def get_data(filters):
+    grand_qty = grand_total = grand_tax_exempt = grand_taxable = grand_tax = 0.0
 
+    for inv in invoices:
         item_filters = {"parent": inv.invoice}
         items = frappe.get_all("Sales Invoice Item", filters=item_filters,
-            fields=["is_nontaxable_item", "net_amount", "amount", "item_code", "qty", "uom", "item_name"])
+            fields=["is_nontaxable_item", "net_amount", "amount", "item_code", "qty", "uom", "item_name", "item_tax_template"])
 
         item_codes = [item["item_code"] for item in items]
         asset_items = frappe.get_all("Item", filters={"item_code": ["in", item_codes], "is_fixed_asset": 1}, pluck="item_code")
 
+        total_invoice = tax_exempt_total = taxable_total = total_qty = 0.0
```

Ruff (0.12.2)

Local variable `total_invoice` is assigned to but never used

Remove assignment to unused variable `total_invoice`

(F841)